src/commands/device/ssh.ts

```diff
@@ -82,7 +82,7 @@ export default class DeviceSSHCmd extends Command {
 				SSH server port number (default 22222) if the target is an IP address or .local
 				hostname. Otherwise, port number for the balenaCloud gateway (default 22).`,
 			char: 'p',
-			parse: async (p) => parseAsInteger(p, 'port'),
+			parse: async (p) => Promise.resolve(parseAsInteger(p, 'port')),
```

Comment:
nit: I see you kept the `async` here but removed it in other spots. We should pick and stick to one :)

Reply:
But do we even need the `Promise.resolve` in all these cases? Does `parse` require a promise? I would assume we can just remove the `async` and not have a `Promise.resolve` at all

Reply:
The expected and enforced interface by oclif is to return a promise. But I agree, the Promise.resolve should be inside the parseAsInteger function and any other parse function. Fixed.